src/set_theory/ordinal_arithmetic.lean

```diff
@@ -371,6 +371,12 @@ theorem is_normal.strict_mono {f} (H : is_normal f) : strict_mono f :=
   (λ b l IH h, lt_of_lt_of_le (H.1 a)
     ((H.2 _ l _).1 le_rfl _ (l.2 _ h)))
 
+theorem is_normal_iff_strict_mono_limit (f : ordinal → ordinal) :
```

Comment:
The name and the statement disagree about which direction the `iff` goes. Please swap the LHS and RHS, are swap the two sides of the name.